test/it/ServicesISpec.scala

```diff
@@ -45,12 +45,12 @@ class ServicesISpec extends PlaySpec with GuiceOneAppPerSuite with TryValues {
 
       s"enforce statement not contain missing milestones for $service" in {
         val statement: AccessibilityStatement = statementTry.get
-        val hasMilestones = statement.milestones.getOrElse(Seq.empty).nonEmpty
+        val hasMilestones                     = statement.milestones.getOrElse(Seq.empty).nonEmpty
         hasMilestones || statement.isNonCompliant || statement.isFullyCompliant must be(true)
       }
 
       s"enforce serviceDomain in the format of aaaa.bbbb.cccc for $service" in {
-        val domainRegex = "([a-z0-9-]*[\\.]*)*[a-z0-9]*"
+        val domainRegex                       = "([a-z0-9-]*[\\.]*)*[a-z0-9]*"
```

Comment:
We've got quite a lot of commits which change the formatting of spaces.  Can we look into standardising this please?

Reply:
It is standardised through /.scalafmt.conf, I recommend installing the IDE plugin so it auto-saves.